packages/components/src/filters/advanced/test/utils.js

```diff
@@ -0,0 +1,83 @@
+/**
+ * Internal dependencies
+ */
+import { textContent } from '../utils';
+
+describe( 'textContent()', () => {
+	test( 'should be got text `Hello World`', () => {
```

Comment:
`should be got text` => `should get text`

Reply:
Heh, copy pasta'd from https://github.com/rwu823/react-addons-text-content/blob/master/test/react-addons-text-content.spec.js#L5.

I'll fix it.

Reply:
Fixed in 9ce4d9e.